src/components/anon-menu.vue

```diff
@@ -40,7 +40,6 @@ export default defineComponent({
     return {
         login() {
             store.dispatch("auth/openLogin");
-            popoverController.dismiss();
```

Comment:
Would it not be sufficient to switch these two instead: first close the popover, then trigger the dispatch? Rather than this complicated check for platform below?

Or does that still show the problems?

if not, just  switch the order and put inline comments in there explaining that the order matters, referencing the issue.

Reply:
I believe there will be simpler ways to fix the problem.  Changing the two causes the error to remain, I think the error may be related to dispatch() after loading the "Login" component. This is strange, it is as if the component is called two times.

I haven't sinced the whole application yet, and this leaves me without confidence to make deeper modifications, for now this way works and we can proceed until we find another alternative, what do you think?
Cause error: 

![Captura de ecrã 2021-04-24, às 15 00 15](https://user-images.githubusercontent.com/31784632/115961605-057b6f00-a50f-11eb-9b20-fbcb0ce349c8.png)